cli/azd/pkg/input/progress_log.go

```diff
@@ -118,27 +118,29 @@ func (p *progressLog) Start() {
 
 // Stop clears the screen from any previous output and clear the buffer.
```

Comment:
I think there's several places in this file where comments are made outdated by the change here.